psydac/api/feec.py

```diff
@@ -342,7 +344,7 @@ def conforming_projectors(self, kind='femlinop', p_moments=-1, hom_bc=False):
         if kind == 'femlinop':
             return cP0, cP1, FemLinearOperator(fem_domain=self.V2, fem_codomain=self.V2, linop=cP2, sparse_matrix=cP2.tosparse)
         elif kind == 'sparse':
-            return cP0.tosparse, cP1.tosparse, cP2.tosparse
+            return cP0.tosparse, cP1.tosparse, cP2.tosparse()
```

Comment:
why adding parentheses for cP2 and not for the others ?

Reply:
Because at this point `cP2` is a `IdentityLinearOperator`, which doesn't have `tosparse()` as a property. I changed it a little to make it more consistent.